multiaccounts/accounts/node_config.go

```diff
@@ -0,0 +1,711 @@
+package accounts
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/p2p/discv5"
+	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/status-im/status-go/params"
+	"github.com/status-im/status-go/rpc/network"
+	"github.com/status-im/status-go/sqlite"
+)
+
+const StaticNodes = "static"
+const BootNodes = "boot"
+const TrustedMailServers = "trusted_mailserver"
+const PushNotificationsServers = "pushnotification"
+const RendezvousNodes = "rendezvous"
+const RelayNodes = "relay"
+const StoreNodes = "store"
+const FilterNodes = "filter"
+const LightpushNodes = "lightpush"
+const WakuRendezvousNodes = "waku_rendezvous"
+const DiscV5BootstrapNodes = "discV5boot"
+
+func (db *Database) nodeConfigWasMigrated(tx *sql.Tx) (migrated bool, err error) {
+	row := tx.QueryRow("SELECT exists(SELECT 1 FROM node_config)")
+	switch err := row.Scan(&migrated); err {
+	case sql.ErrNoRows:
+		return migrated, nil
+	case nil:
+		return migrated, nil
```

Comment:
It is a matter of preference, but you can group cases in a switch case. If you prefer the original way that's completely cool, but I just thought I'd highlight this other way.

```suggestion
	case sql.ErrNoRows, nil:
		return migrated, nil
```